bitbots_blackboard/src/bitbots_blackboard/capsules/pathfinding_capsule.py

```diff
@@ -17,11 +17,19 @@ def __init__(self):
         self.orientation_threshold = rospy.get_param('behavior/body/pathfinding_orientation_threshold')
         self.pathfinding_pub = None  # type: rospy.Publisher
         self.pathfinding_cancel_pub = None  # type: rospy.Publisher
+        self.ball_obstacle_active_pub = None
+        self.approach_marker_pub = None
         self.goal = None  # type: PoseStamped
         self.current_pose = None # type: PoseStamped
+        self.status = -1 # Current status of movebase
+        self.ACTIVE = 1
+        self.CANCELED = 2
+        self.SUCCEEDED = 3
+        self.avoid_ball = True
```

Comment:
Not sure whether we want this to be true by default. I would prefer that we need to actively decide to avoid it.

Reply:
Yea but we also avoid all other obstacles by default, so I would define this as an explicit opt-out operation.